src/strategies/ocean-marketplace/examples.json

```
@@ -1,22 +1,41 @@
 [
   {
-    "name": "OCEAN tokens staked on the marketplace",
+    "name": "ERC-20 OCEAN tokens staked on the marketplace",
     "strategy": {
       "name": "ocean-marketplace",
       "params": {
-        "address": "0x967da4048cD07aB37855c090aAF366e4ce1b9F48",
-        "symbol": "OCEAN",
-        "decimals": 18
+        "expectedResults": {
+          "scores": {
+            "0x655eFe6Eb2021b8CEfE22794d90293aeC37bb325": 234872.06400571144,
+            "0xCe7BE31f48205C48A91A84E777a66252Bba87F0b": 184463.92011443214,
+            "0x788550D00579F66c06ce209D14056C8F2c0A8188": 7288.420566804952,
+            "0x18F93FA526e598769fd909F1D2C82315219d49Ed": 6163.508200256553,
+            "0xA321025df54C18CEeFdE118c0671735390aCB318": 4532.791480521258,
+            "0x8a4C36a99b0418BF95A155160C31A267EBCA8754": 3627.4502012669805,
+            "0x5e22b34F2708EA7e25918B8FCF816F9CfF27a331": 4460.415948645305,
+            "0x5266dc406d688bcb152bfCf2C412113b821c5A43": 2893.9739596759805,
+            "0x620bDD183Ec570E4de54001A6e3CbF4215d6d326": 3140.1538635109296,
+            "0xB9b14C4d0cbc18FC5f54701D578e85968cf19FD5": 2706.7321282426365
+          }
+        }
       }
     },
     "network": "1",
     "addresses": [
       "0x655eFe6Eb2021b8CEfE22794d90293aeC37bb325",
+      "0xCe7BE31f48205C48A91A84E777a66252Bba87F0b",
+      "0x788550D00579F66c06ce209D14056C8F2c0A8188",
+      "0x18F93FA526e598769fd909F1D2C82315219d49Ed",
+      "0xA321025df54C18CEeFdE118c0671735390aCB318",
+      "0x8a4C36a99b0418BF95A155160C31A267EBCA8754",
+      "0x5e22b34F2708EA7e25918B8FCF816F9CfF27a331",
+      "0x5266dc406d688bcb152bfCf2C412113b821c5A43",
+      "0x620bDD183Ec570E4de54001A6e3CbF4215d6d326",
+      "0xB9b14C4d0cbc18FC5f54701D578e85968cf19FD5",
       "0x12BD31628075C20919BA838b89F414241b8c4869",
       "0xf88bD9c3F81f7148E4076bBB8b2e2B0951A8cE38",
       "0xAAB9EaBa1AA2653c1Dda9846334700b9F5e14E44"
     ],
-    "snapshot": "11635000"
+    "snapshot": "12097120"
```

This should be a number
```suggestion
    "snapshot": 12097120
```